### PR TITLE
Deploy dev to pypi from travis

### PR DESCRIPTION
**Pull request by keflavich**
cc @bsipocz.  See #1115.

### CODE REVIEW

**Comment by keflavich** [comment]
How do we test this?

File: .travis.yml [hunk 1]
```diff
@@ -148,6 +150,16 @@ matrix:
           env: MAIN_CMD='pycodestyle astroquery --count'
                SETUP_CMD='' EVENT_TYPE='push pull_request'
 
+        - stage: deploy
+          deploy:
+              provider: pypi
+              user: "keflavich"
```
Comment (bsipocz) [inline .travis.yml]: do we want maybe to set up an auto deploy user that will only have access to astroquery rather your whole pypi?

**Comment by bsipocz** [comment]
@keflavich - I'm not sure, the best test probably to try out. There is a PR suitable to be merged right now, so we can use it. 
Also, probably we'll need fine tuning, as currently the deploy stage won't run as master is always failing with the remote ones. But that's just a shuffle around jobs.

File: .travis.yml (hunk 1 above)
Comment (keflavich) [reply .travis.yml]: Could do that for sure.  I guess we can just set up a user called 'astroquery'?

**Pull request merged**